python/fastquant/strategies/base.py

```diff
@@ -46,14 +46,34 @@ class BaseStrategy(bt.Strategy):
             "execution_type",
             "close",
         ),  # Either open or close, to indicate if a purchase is executed based on the next open or close
-        ("periodic_logging", False),
+        ("periodic_logging", False), #DO NOT set to True else return_transactions
         ("transaction_logging", True),
     )
 
     def log(self, txt, dt=None):
+        """
+        FIXME: parsing data from txt assumes a format
+        """
```

Comment:
This is just a hack to parse timestamps from logs instead of unearthing stuff from `strat` instance

Reply:
Actually, my preference is to get this properly done straight from the strat object. Will chime in to figure this out sometime tonight.

Reply:
I agree we should access data from `strat` but for now I found parsing filtered logs easier.